Packages/Editor/Sources/Editor/EditorViewController.swift

```diff
@@ -182,6 +190,18 @@ public final class EditorViewController: UIViewController {
         navigationController?.pushViewController(previewController, animated: true)
     }
 
+    @objc
+    private func showPhotoPicker() {
+        textView.resignFirstResponder()
+        var configuration = PHPickerConfiguration()
+        configuration.filter = .images
+        configuration.selectionLimit = 1
+        configuration.preferredAssetRepresentationMode = .automatic
+        let picker = PHPickerViewController(configuration: configuration)
+        picker.delegate = self
+        present(picker, animated: true)
```

Comment:
Question: Can we support adjusting the image like crop or something by PhotosUI?